React/Views/ScrollView/RCTScrollView.m

```diff
@@ -760,13 +761,18 @@ - (void)viewDidMoveToWindow
   if ([self window] == nil) {
     // Unregister for bounds change notifications
     [[NSNotificationCenter defaultCenter] removeObserver:self name:NSViewBoundsDidChangeNotification object:_scrollView.contentView];
+    [[NSNotificationCenter defaultCenter] removeObserver:self name:NSPreferredScrollerStyleDidChangeNotification object:nil];
   }
   else {
     // Register for bounds change notifications so we can track scrolling
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(scrollViewDocumentViewBoundsDidChange:)
                                                  name:NSViewBoundsDidChangeNotification
                                                object:_scrollView.contentView];  // NSClipView
+    [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Another case here and above where we can avoid trivial inefficiency by saving defaultCenter to a local